aws/eks/cloudwatch_queries.tf

```
@@ -11,7 +11,7 @@ resource "aws_cloudwatch_query_definition" "celery-errors" {
   query_string = <<QUERY
 fields @timestamp, log, kubernetes.container_name as app, kubernetes.pod_name as pod_name, @logStream
 | filter kubernetes.container_name like /^celery/
-| filter @message like /ERROR\/.*Worker/
+| filter @message like /ERROR\/.*Worker/ or @message like /ERROR\/ForkPoolWorker/ or @message like /WorkerLostError/ or @message like /ERROR\/MainProcess/ 
```

We'd just need that though, no?

```suggestion
| filter @message like /ERROR\/.*Worker/ or  @message like /ERROR\/MainProcess/ 
```

---

We had different selectors in different places (query, dashboard, alarm) so I thought I'd make them all the same (ie add all the different types of errors that appeared in all the filters)

---

But would not `/ERROR\/.*Worker/` already catch `@message like /ERROR\/ForkPoolWorker/ or @message like /WorkerLostError/`? The query execution engine might never get to the last conditions because the first filter would be a positive match. The only new element would be `@message like /ERROR\/MainProcess/` if I am not mistaken.

---

oh yes I see what you're saying. Ah yeah this is a good point, yes 💯 . Let me simplify things!